live-examples/html-examples/content-sectioning/hgroup.html

```diff
@@ -1,5 +1,5 @@
 <hgroup>
-    <h1>Calculus I</h1>
-    <p>Fundamentals</p>
+	<h1>Frankenstein</h1>
+	<p>Or: The Modern Prometheus</p>
 </hgroup>
-<p>This course will start with a brief introduction about the limit of a function. Then we will describe how the idea of derivative emerges in the Physics and Geometry fields. After that, we will explain that the key to master calculus is &hellip;</p>
+<p>Victor Frankenstein, a Swiss scientist, has a great ambition: to create intelligent life. But when his creature first stirs, he realizes he has made a monster. A monster which, abandoned by his master and shunned by everyone who sees it, follows Dr Frankenstein with murder and horrors to the very ends of the earth.</p>
```

Comment:
```suggestion
<p>Victor Frankenstein, a Swiss scientist, has a great ambition: to create intelligent life. But when his creature first stirs, he realizes he has made a monster. A monster which, abandoned by his master and shunned by everyone who sees it, follows Dr Frankenstein to the very ends of the earth.</p>
```

(perhaps I am over-sensitive)

Reply:
It's copy-paste of official description, but sure.